docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -217,6 +217,13 @@ Enable monitor status alerts.
 +
 `tls.enabled` (`boolean`):::
 Enable TLS certificate alerts.
+
+`retestOnFailure` (`boolean`)::
+Enable or disable retesting when a monitor fails. Default is `true`.
++
+By default, monitors are automatically retested if the monitor goes from an "up" to "down" condition.
```

Comment:
I think this is clear, but do you think it's worth reinforcing this, so it's clear retests won't happen if the monitor was already down?

e.g.
- scheduled_test (UP)
- scheduled_test (DOWN)
- retest (DOWN)
- scheduled_test (DOWN)
- scheduled_test (DOWN)
i.e. it won't continually retest for every DOWN

Reply:
@elastic/obs-docs I'm curious what you think. To me, _goes from "up" to "down"_ makes it clear that it's only when "down" comes immediately after an "up", but if there is a way to make it clearer I'm open to suggestions!

Reply:
@colleenmcginnis The ambiguous part for me is: what happens after the monitor goes "down" and the error is generated? It sounds like the monitor is tested again at the scheduled time (rather than being retesting over and over again until the monitor is back "up"). If this is the case, it wouldn't hurt to add another sentence that clarifies.

Reply:
You could add something like: " The monitor will be tested again at the scheduled time."